apps/src/templates/teacherNavigation/PageHeader.tsx

```diff
@@ -28,7 +34,24 @@ const PageHeader: React.FC = () => {
   const isLoadingSectionData = useAppSelector(
     state => state.teacherSections.isLoadingSectionData
   );
+  const [ageGatedModalOpen, setAgeGatedModalOpen] = useState(false);
+  const toggleAgeGatedModal = () => {
+    setAgeGatedModalOpen(!ageGatedModalOpen);
+  };
   const selectedSection = useAppSelector(selectedSectionSelector);
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    dispatch(loadSectionStudentData(selectedSection.id));
```

Comment:
This is also called from the `manageStudents` component but should load into the same redux object. Could we simplify by also removing from ManageStudents?

Assuming we don't take the BE approach below

Reply:
Nice catch! I have removed the redundant call.